source/scripts/app.js

```diff
@@ -51,6 +105,54 @@ function createExplorePage() {
     "recipe-cards-section"
   );
 
+  //Get references to search bar on explore
+  let shadow = explorePage.shadowRoot;
+  let input = shadow.getElementById("search-bar");
+  let vegan = shadow.getElementById("vegan");
+  let glutenFree = shadow.getElementById("gluten-free");
+  let vegetarian = shadow.getElementById("vegetarian");
+  input.addEventListener("keyup", (e) => {
```

Comment:
Make this an `async` arrow function
```suggestion
  input.addEventListener("keyup", async (e) => {
```